docs/validators/node-upgrades.md

```diff
@@ -0,0 +1,73 @@
+---
+id: node-upgrades
+title: Elrond Node upgrades
+---
+
+## **Introduction**
+
+Once a new node's binary is ready to be deployed on one of a network (mainnet, testnet or devnet), nodes operators must 
+perform the upgrade to the newest version. These releases are always announced on Elrond [Validators chat](https://t.me/ElrondValidators) 
+plus via other communication channels, depending on the case. 
+
+## **Types of upgrades**
+
+Currently, we have the following types of upgrades:
+
+A. - **all nodes need to upgrade**:  upgrades that involve processing changes with an activation epoch (as explained below) 
+and have to be performed by all nodes operators in order to keep the same view over the network and not cause service disruptions.
+
+B. - **optional upgrades**: upgrades that for example simply add a new Rest API endpoint or improve the trie syncing timing 
+are not critical from a processing point of view, and they are optional. If the nodes operators think the new feature will help them,
+they can proceed with the upgrade without losing the compatibility with the network.
```

Comment:
Some newlines are slightly unexpected (in the markdown source, not necessarily in the rendered output).